File: gluecodium/src/test/resources/smoke/skip/output/android/com/example/smoke/EnableIfEnabled.java
/*
 *
 */
package com.example.smoke;
import com.example.NativeBase;
public final class EnableIfEnabled extends NativeBase {
    /**
     * For internal use only.
     * @hidden
     */
    protected EnableIfEnabled(final long nativeHandle, final Object dummy) {
        super(nativeHandle, new Disposer() {
            @Override
            public void disposeNative(long handle) {
                disposeNativeHandle(handle);
            }
        });
    }
    private static native void disposeNativeHandle(long nativeHandle);
    public static native void enableIfUnquoted();
    public static native void enableIfUnquotedList();
    public static native void enableIfQuoted();
    public static native void enableIfQuotedList();
    public static native void enableIfTagged();
    public static native void enableIfTaggedList();
    public static native void enableIfMixedList();
}

File: gluecodium/src/test/resources/smoke/skip/output/android-kotlin/com/example/smoke/EnableIfEnabled.kt
/*

 *
 */

@file:JvmName("EnableIfEnabled")

package com.example.smoke

import com.example.NativeBase

class EnableIfEnabled : NativeBase {



    /*
     * For internal use only.
     * @hidden
     * @param nativeHandle The handle to resources on C++ side.
     * @param tag Tag used by callers to avoid overload resolution problems.
     */
    protected constructor(nativeHandle: Long, tag: Any?)
        : super(nativeHandle, { disposeNativeHandle(it) }) {}







    companion object {
        @JvmStatic private external fun disposeNativeHandle(nativeHandle: Long)
        @JvmStatic external fun enableIfUnquoted() : Unit
        @JvmStatic external fun enableIfUnquotedList() : Unit
        @JvmStatic external fun enableIfQuoted() : Unit
        @JvmStatic external fun enableIfQuotedList() : Unit
        @JvmStatic external fun enableIfTagged() : Unit
        @JvmStatic external fun enableIfTaggedList() : Unit
        @JvmStatic external fun enableIfMixedList() : Unit
    }
}
